File: Part 1- Data Preprocessing/section 2/data_preprocessing_template.R
#Importing the dataset
dataset = read.csv('Data.csv')
#missing dataset


#encoding categorical data


#splitting dataset
set.seed(123)
split = sample.split(dataset$Purchased, SplitRatio = 0.8) #for training set
training_set = subset(dataset, split == TRUE)
test_set = subset(dataset, split == FALSE)

#feature scaling
# training_set[,2:3] = scale(training_set[,2:3])
# test_set[, 2:3] = scale(test_set[,2:3])

File: Part 1- Data Preprocessing/section 2/data_preprocessing_template.py
import numpy as np #for math
import matplotlib.pyplot as plt # for plotting
import pandas as pd # import and manage dataset

#import dataset
dataset = pd.read_csv('Data.csv')
X = dataset.iloc[:, :-1].values
Y = dataset.iloc[:, 3].values
# missing data


#encoding categorical data


#splitting data
from sklearn.model_selection import train_test_split
X_train, X_test, Y_train, Y_test = train_test_split(X, Y, test_size = 0.2, random_state = 0)

#feature scaling
"""from sklearn.preprocessing import StandardScaler
sc_X = StandardScaler()
X_train = sc_X.fit_transform(X_train)
X_test = sc_X.transform(X_test)"""

